dev/devfile.yaml

```diff
@@ -10,28 +10,25 @@ projects:
       location: 'https://github.com/OpenLiberty/application-stack.git'
       sparseCheckoutDir: /templates/default
 components:
-  - id: redhat/java/latest
-    memoryLimit: 1512Mi
-    type: chePlugin
-  - alias: devruntime
-    mountSources: true
-    type: dockerimage
-    image: kabanero/ubi8-maven:0.3.1 
-    memoryLimit: 1512Mi
-    volumes:
-    - name: m2
-      containerPath: /home/user/.m2
-    endpoints:
-    - name: 9080/tcp
-      port: 9080
-    - name: 9443/tcp
-      port: 9443
+- type: chePlugin
+  id: redhat/java/latest
+  memoryLimit: 1512Mi
+- alias: devruntime
+  mountSources: true
+  type: dockerimage
```

Comment:
Let's add a comment pointing to the source for this image.

I'd like to move the source to our stack.  We're not getting much from being in the devfile registry repo.